ir/src/constraints/graph.rs

```diff
@@ -426,6 +434,9 @@ impl AlgebraicGraph {
             IdentifierType::PublicInput(_) => {
                 unimplemented!("TODO: add support for public inputs.")
             }
+            IdentifierType::RandomValue(offset, _size) => {
```

Comment:
minor nit: @grjte pointed this out elsewhere that it would be better to do RandomValue(offset, _)